brave/common/extensions/url_bindings.cc

```diff
@@ -300,6 +304,13 @@ void URLBindings::New(const v8::FunctionCallbackInfo<v8::Value>& args) {
 
 void URLBindings::FormatForDisplay(
     const v8::FunctionCallbackInfo<v8::Value>& args) {
+  this->FormatForDisplay(args, net::UnescapeRule::NONE);
+}
+
+void URLBindings::FormatForDisplay(
```

Comment:
I left the changes as C++ only because I'm not sure of specifics, like those. By doing the static_cast above, I'm forcing the version without args to be used.  This does work great.  If we want to expose the UnescapeRule, I'm not sure how to do that properly